### PR TITLE
Patch version of TAEF referenced by DevCheck

### PR DESCRIPTION
**Pull request by DrusTheAxe**
Quick patch to unblock folks.

More dynamic update using Version.Dependencies.xml info coming in a future PR

### CODE REVIEW

**Review by pmpurifoy — APPROVED** [review]

**Comment by DrusTheAxe** [comment]
Quick patch to unblock folks.

More dynamic update using Version.Dependencies.xml info coming in a future PR

File: DevCheck.ps1 [hunk 1]
```diff
@@ -605,7 +605,7 @@ function Repair-TAEFService
 
     $root = Get-ProjectRoot
     $cpu = Get-CpuArchitecture
-    $taef = 'Microsoft.Taef.10.58.210222006-develop'
+    $taef = 'Microsoft.Taef.10.75.221207001'
```
Comment (bpulliam) [inline DevCheck.ps1:608]: Why not keep this value in Version.Details.xml?

Comment (DrusTheAxe) [reply DevCheck.ps1:608]: Time.

This is just a quick'n'dirty patch to unblock folks. Another (more complex) PR will use Version.Dependencies.xml, as you'd expect

Patience please. Rome wasn't built in a fortnight :P

**Review by bpulliam — APPROVED** [review]

**Pull request merged**